test/WebJobs.Script.Tests/MetricsEventManagerTests.cs

```diff
@@ -72,20 +72,6 @@ public class MetricsEventManagerTests
                 string.Format("There are events with different execution id. List:{0} Invalid entries:{1}",
                     SerializeFunctionExecutionEventArguments(argsList),
                     SerializeFunctionExecutionEventArguments(invalidArgsList)));
-
```

Comment:
I thought we were in hurry to release this tomorrow :). If that is not the case then I would fix this tomorrow morning.